Packs/CyrenInboxSecurity/README.md

```diff
@@ -0,0 +1,98 @@
+# Cyren Inbox Security
+
+
+
+Note: This is a beta pack, which lets you implement and test pre-release software. Since the pack is beta, it might contain bugs. Updates to the pack during the beta phase might include non-backward compatible features. We appreciate your feedback on the quality and usability of the pack to help us identify issues, fix them, and continually improve.
```

Comment:
@eran-levy @ditroia - Is this a beta pack? If so, what is the reason for this?

Reply:
removed. done.